waspc/cli/Wasp/Cli/Command/Db/Migrate.hs

```diff
@@ -21,8 +21,8 @@ import qualified Wasp.Generator.DbGenerator.Operations as DbOps
 -- | NOTE(shayne): Performs database schema migration (based on current schema) in the generated project.
 -- This assumes the wasp project migrations dir was copied from wasp source project by a previous compile.
 -- The migrate function takes care of copying migrations from the generated project back to the source code.
-migrateDev :: Command ()
-migrateDev = do
+migrateDev :: Maybe String -> Command ()
+migrateDev maybeMigrationName = do
```

Comment:
Would it be possible to require the name here and to ensure higher up that a default name is supplied if not given? Maybe that's tricky to accomplish?

Reply:
I'm not sure we want to use a default name. Either they gave one to us, or we let Prisma ask them interactively (current default behavior).

Reply:
I agree with this for now, to not use a default name.

Reply:
yeah, as @shayneczyzewski said, it seems that Prisma's way of working is not having default names, as it kinda encourages bad practices (losing info, I guess). So I think it will not be strange if we follow that.